pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -81,6 +86,48 @@ func podHasPVCs(pod *v1.Pod) bool {
 	return false
 }
 
+// PreFilter invoked at the prefilter extension point to check if pod has all
+// immediate PVCs bound. If not all immediate PVCs are bound, an
+// UnschedulableAndUnresolvable is returned.
+func (pl *VolumeBinding) PreFilter(ctx context.Context, state *framework.CycleState, pod *v1.Pod) *framework.Status {
```

Comment:
Can we add unit tests for these new states and status?

Reply:
pushed a new commit https://github.com/kubernetes/kubernetes/pull/91775/commits/67c9ed7f4ee328a4c13fcf5f51fefbbc8ea84800

I removed the usage of FakeVolumeBinder. I think it's better to use fake kubernetes client with real volume binder 
I also removed some unnecessary tests and add new tests for `PreFilter` statues and states.